WooCommerce/Classes/Analytics/WooAnalytics.swift

```diff
@@ -143,10 +144,25 @@ public extension WooAnalytics {
         guard let error = error else {
             return nil
         }
+
+        var errorCode = ""
+        var errorDomain = ""
+        var errorDescription = ""
```

Comment:
Super nit: to avoid having to initiate all these as variables, we can clean up a bit:
```suggestion
        let err = error as NSError
        let errorCode: String = {
            if let networkError = error as? AFError {
                return "\(networkError.responseCode ?? 0)"
            }
            return "\(err.code)"
        }()
        let errorDomain = err.domain
        let errorDescription = err.description
```

Reply:
TIL! Changed on https://github.com/woocommerce/woocommerce-ios/pull/8789/commits/5015943b07485681f3308f97906a0bf4180d571c